fabrics.c

```diff
@@ -730,7 +739,7 @@ int nvmf_discover(const char *desc, int argc, char **argv, bool connect)
 	if (context)
 		nvme_root_set_application(r, context);
 
-	if (!nvme_read_config(r, config_file))
```

Comment:
Seems okay to check the errno value as below.
```c
    errno = 0;
    if (!nvme_read_config(r, config_file) && !errno)
```

Reply:
`nvme_read_config` should actually not touch errno if there is failure. This is actually a bug in libnvme... Good catch!

Reply:
Yes agreed. Thank you.